src/System.Memory/ref/System.Memory.cs

```diff
@@ -281,6 +281,66 @@ public static class BinaryPrimitives
 
 namespace System.Buffers.Text
 {
+    public struct StandardFormat : IEquatable<StandardFormat>
```

Comment:
I think we talked about this type being in System.Buffers namespace. I know that SignalR team wanted to use it as a format specifier for non-textual output formats.

Reply:
Done